packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -53,17 +54,22 @@ export type TabsProps = {
    */
   ariaLabel?: string;
   /**
-   * The value of the Tab that should be selected by default
+   * @deprecated please use value instead
```

Comment:
```suggestion
   * @deprecated please use the `value` prop instead.
```

I'm formatting this because VS Code displays Markdown in the mouse-over pop-up.

And I want it more clear what this means. People will say "what value?" I want them to know it's a prop.